### PR TITLE
docs: debugging

### PR DESCRIPTION
**Pull request by p-runge**
tng-1024

### CODE REVIEW

**Review by henczi-espirit — COMMENTED** [review]
should this be checked in? (is it on purpose?)

**Review by henczi-espirit — CHANGES_REQUESTED** [review]
Please remove the launch.json, fix the typo in the doc and decide if you want to have the "debugger" keyword part of the documentation.

File: docs/modules/ROOT/pages/Debugging.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
+= Debugging
+
+== Preparations
+
+=== Download EDI
```
[the remark this answers is not in the document]
Comment (p-runge) [reply docs/modules/ROOT/pages/Debugging.adoc:5]: Fixed

File: docs/modules/ROOT/pages/Debugging.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "webRoot": "${workspaceFolder}",
+      "devtool": "source-map"
+    }
+  ]
+}
+```
+
+Keep in mind to change the port in the url if using another port. After that you can start the application in the default terminal.
+
+Click on the launch button at the bottom bar of VS Code to launch a Chrome window where the debugger is attached to. You can set breakpoints wherever you want to debug your code by either toggling a red dot on the left side of each line by clicking there, or by using the `debugger` keyword in your code.
```
[the remark this answers is not in the document]
Comment (p-runge) [reply docs/modules/ROOT/pages/Debugging.adoc:38]: I assumed having a linter in the CI that wouldn't allow pushing those things. Nevertheless it's not really needed anyway, so I'll just erase it.

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (p-runge) [inline .vscode/launch.json:1]: I'll add the whole .vscode directory to .gitignore since it seems to have been pushed by mistake initially.

**Review by henczi-espirit — APPROVED** [review]

**Pull request merged**